onedal/basic_statistics/basic_statistics.py

```diff
@@ -17,8 +17,11 @@
 from sklearn.base import BaseEstimator
 from abc import ABCMeta, abstractmethod
 
-import numpy as np
-from numbers import Number
+# TODO:
+# this interface will be wrapped in _data_conversion module.
+import dpctl.tensor as dpt
+
+import dpnp
```

Comment:
What about just importing `dpnp` as `np`?

Reply:
1) dpnp doesn't support all numpy functionality.

2) This is draft code. I will moved explicit data frame call into separate module